tests/integration/view-change.test.js

```diff
@@ -106,7 +117,13 @@ describe('view change', function () {
 
                 const sortedDaemons = orderBy(this.swarm.getDaemons(), ['publicKey']);
 
-                const statusResponse = await pTimeout(this.api.status(), 4500, 'Status() failed to respond in 4500ms');
+                let statusResponse;
+
+                try {
+                    statusResponse = await this.api.status();
+                } catch (err) {
+                    log.crit(err);
```

Comment:
Since this continues with `statusResponse` being `undefined`, the next statement will throw `TypeError`.

Reply:
Fixed